docs/eigenda/operator-guides/run-a-node/registration.mdx

```diff
@@ -73,7 +73,8 @@ More information about the registration process that is executed by the above co
 
 ## Check for network traffic
 
-Once you have successfully opted into a quorum, you should begin to see logs indicating that your node is receiving, validating, and storing batches from the network, like the following:
+EigenDA uses the operator state that is 75 blocks (15 minutes) behind the current chain head to ensure the state is not at risk of being reorg'd.
```

Comment:
Having the parameter hard-coded into the doc probably isn't ideal I guess. But this seems fine for now, we can clean up later.

Reply:
+1 I don't know if there's a way to load these parameters. 
We also have avssync interval (6h) hardcoded in L60